Service/AtendimentoService.php

```diff
@@ -149,8 +149,8 @@ public function chamarSenha(Unidade $unidade, Atendimento $atendimento)
         $senha->setSiglaSenha($atendimento->getSenha()->getSigla());
         $senha->setMensagem($su->getMensagem() . '');
         // local
-        $senha->setLocal($atendimento->getLocal()->getNome());
-        $senha->setNumeroLocal($atendimento->getNumeroLocal());
+        $senha->setLocal($su->getLocal()->getNome());
+        $senha->setNumeroLocal($atendimento->getLocal());
```

Comment:
why are you setting `?Local`  to `?int`?

https://github.com/novosga/core/blob/master/Entity/AbstractAtendimento.php#L207-L210
https://github.com/novosga/core/blob/master/Entity/PainelSenha.php#L56-L59